operators/multiclusterobservability/manifests/base/grafana/dash-acm-openshift-virtualization-overview.yaml

```diff
@@ -21,44 +21,41 @@ data:
           }
         ]
       },
-      "description": "This dashboard shows details related to the OpenShift Virtualization operator and Virtual machines",
+      "description": "This dashboard provides insights into the health of the OpenShift Virtualization operator and displays key metrics for virtual machines at the cluster level across multiple clusters. It helps monitor and assess the status of virtualization resources, ensuring smooth operation and quick identification of issues.",
       "editable": true,
       "gnetId": null,
       "graphTooltip": 0,
-      "id": 29,
-      "uid": "OmpD1ZoSk",
-      "iteration": 1709210609633,
+      "id": 45,
```

Comment:
One note is all the dashboards need a unique UUID as per this PR
https://github.com/stolostron/multicluster-observability-operator/pull/1566/files if they ever link back to themselves in the dashboard definition.

Reply:
All dashboards have a uid